docs/DeveloperGuide.adoc

```diff
@@ -774,7 +1074,25 @@ Use case resumes at step 2.
 
 .  Should work on any link:#mainstream-os[mainstream OS] as long as it has Java `1.8.0_60` or higher installed.
 .  Should be able to hold up to 1000 persons without a noticeable sluggishness in performance for typical usage.
+.  Should be compatible with Mac and Windows users
```

Comment:
users or operating systems?

Reply:
I think, we mean "both". Sorry for the confusion, will change it now.